src/app/screens/BookDetail/components/BookDetail/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { View, Text, Image } from 'react-native';
+import Wbutton from '../../../../components/Wbutton';
+import styles from './styles';
+import { imageMissing } from '../../../../../constants/constants';
+
+interface Props {
+  title: string;
+  author: string;
+  imageUri: string | null;
+  year: string;
+  genre: string;
+}
```

Comment:
i think since you are using this interface in more than one component, you should extract it into another file. We usually have an interface folder next to the app folder that has all the global interfaces used in the app

Reply:
Done!